[lldb] Suppress unsupported language warning for assembly

JDevlieghere opened the pull request:
The following warning is technically correct, but pretty much useless, since there aren't any frame variables that we'd expect the debugger to understand.

> This version of LLDB has no plugin for the language "assembler".
> Inspection of frame variables will be limited.

This message is useful in the general case but should be suppressed for the "assembler" case.

rdar://92745462

medismailben reviewed (approved):
LGTM but all the formatting changes are a bit distracting, would be nice to be have them in a separate commit.

JDevlieghere committed:
[lldb] Suppress unsupported language warning for assembly #95871

The following warning is technically correct, but pretty much useless,
since there aren't any frame variables that we'd expect the debugger to
understand.

> This version of LLDB has no plugin for the language "assembler".
> Inspection of frame variables will be limited.

This message is useful in the general case but should be suppressed for
the "assembler" case.

rdar://92745462

JDevlieghere committed:
Fix test
